src/mutations/user/utils/updateDbEs.ts

```diff
@@ -0,0 +1,56 @@
+import { isNil, omitBy } from 'lodash'
+
+import logger from 'common/logger'
+import { es, knex } from 'connectors'
+import { GQLUpdateUserInfoInput, UserRole } from 'definitions'
+
+/**
+ * Update user db & es record
+ * @param id: user id
+ * @param data: data to be updated
+ */
+export const updateDbEs = async (
```

Comment:
To me, `utils` is more like a place for functions of conversion, formatting and calculation. I recommend to drop function that causes side effect into atomService. I might put es update function in atomService, and reuse `atomService.update` and `atomService.updateEs` in resolvers.

Reply:
@devformatters updated as you suggest, plz review again 🙏